tsl/test/shared/sql/dist_fetcher_type.sql

```diff
@@ -0,0 +1,29 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+\set ON_ERROR_STOP off
+
+-- Test that we use the correct type of remote data fetcher.
+set timescaledb.remote_data_fetcher = 'auto';
+select 1 x from distinct_on_distributed t1, distinct_on_distributed t2
+where t1.id = t2.id + 1
+limit 1;
+
+set timescaledb.remote_data_fetcher = 'cursor';
+select 1 x from distinct_on_distributed t1, distinct_on_distributed t2
+where t1.id = t2.id + 1
+limit 1;
```

Comment:
Can we have EXPLAIN output show the cursor being used?

Reply:
Currently there is no way to tell from EXPLAIN. You mean, add it to the EXPLAIN output in this PR? I'll try that, I guess it might be useful for debugging performance. Maybe I'll only add the output for verbose mode, and also only for 'cursor' mode. 'rowbyrow' should be somewhat like the default now, and 'cursor' is the slower mode for complex queries. So we can only show this slower mode in EXPLAIN to avoid too much verbosity and test reference changes.